eng/tools/rush-runner/index.js

```diff
@@ -16,6 +16,7 @@ const reducedDependencyTestMatrix = {
     "@azure/identity",
     "@azure/service-bus",
     "@azure/template",
+    "@azure-rest/ai-document-intelligence"
```

Comment:
adding document intelligence to core build list to make sure no regressions due to the custom policy

Reply:
If you think ai-document-intelligence provides additional coverage, we can replace synapse-access-control with it.

Reply:
Browser tests are not enabled for document intelligence yet, so I thought I should keep both.

I mainly added it for this reason.. https://github.com/Azure/azure-sdk-for-js/pull/31908#discussion_r1881056901

Reply:
removing since the policy is not being added